pkgs/os-specific/linux/kernel/manual-config.nix

```
@@ -299,13 +299,15 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.platform kernelPatches
   enableParallelBuilding = true;
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
-  nativeBuildInputs = [ perl bc nettools openssl rsync gmp libmpc mpfr ]
+  nativeBuildInputs = [ perl bc nettools openssl rsync gmp libmpc mpfr buildPackages.gawk ]
```

I think splicing should work here for cross compiling:

```suggestion
  nativeBuildInputs = [ perl bc nettools openssl rsync gmp libmpc mpfr gawk ]
```

---

I've applied your file, but do I need to specify it on the top of the file as a parameter then?

---

Yes.